grammar/mongo.g4

```diff
@@ -1,5 +1,14 @@
 grammar mongo;
```

Comment:
This gives an error:

db.heroes.insert({  "id": 2,  "name": "Batman",  "saying": "I'm Batman"})

Reply:
Thanks! Figured out [the bug](https://github.com/Microsoft/vscode-cosmosdb/pull/460/files/cce9a693f81fcf727762a650d54d0e736cf346ad#diff-3e4e420864699bc495e9daad18f7e036R126).